docs/catalog.md

```diff
@@ -270,13 +270,13 @@ The following is an example of creating a package binding and then getting a 10-
 1. Create a package binding with your API key.
 
   ```
-  $ wsk package bind /whisk.system/weather myWeather --param username 'MY_USERNAME' --param password 'MY_PASSWORD'
+  $ wsk package bind /whisk.system/weather myWeather --param username MY_USERNAME --param password MY_PASSWORD
   ```
 
 2. Invoke the `forecast` action in your package binding to get the weather forecast.
 
   ```
-  $ wsk action invoke myWeather/forecast --blocking --result --param latitude '43.7' --param longitude '-79.4'
+  $ wsk action invoke myWeather/forecast --blocking --result --param latitude "43.7" --param longitude "-79.4"
```

Comment:
Went back and forth on this one, wanted to articulate that action expected strings and not interpreted as numbers.
I can remove the quotes, no problem.